libraries/mediaplayer/impl/src/main/kotlin/io/element/android/libraries/mediaplayer/impl/SimplePlayer.kt

```diff
@@ -77,6 +81,10 @@ class SimplePlayerImpl(
     override val duration: Long
         get() = p.duration
 
+    override var playWhenReady: Boolean
+        get() = p.playWhenReady
+        set(value) { p.playWhenReady = value }
```

Comment:
I added this as I felt the API was a bit more explicit (because `play()` only works when ready). But on second thought, I think it goes against the philosophy of the 'simple' player interface! I'll remove.

About the statefulness, FYI [the docs](https://developer.android.com/reference/androidx/media3/common/Player#play()) mention that `play()` and `setPlayWhenReady(true)` are equivalent so I assume `play()` is also stateful. Not sure if that changes any of our assumptions?

Reply:
Removed in https://github.com/vector-im/element-x-android/pull/1758/commits/3febc21abf99daed01e0fa4f494ee2340d85c4ae